### PR TITLE
Update link to official repository 

### PR DESCRIPTION
**Pull request by tomwarwick**

### CODE REVIEW

File: Documentation/BugfixingAZ/Index.rst [hunk 1]
```diff
@@ -91,7 +91,7 @@ and setting up a working TYPO3 installation for core development, skip to number
 
    Push to Gerrit (replace `<YOUR_TYPO3_USERNAME>` here)::
 
-      git config url."ssh://<YOUR_TYPO3_USERNAME>@review.typo3.org:29418".pushInsteadOf git://git.typo3.org
+      git config url."ssh://<YOUR_TYPO3_USERNAME>@review.typo3.org:29418/Packages/TYPO3.CMS.git"
```
Comment (susannemoog) [inline Documentation/BugfixingAZ/Index.rst:94]: git config remote.origin.pushurl ssh://<username>@review.typo3.org:29418/Packages/TYPO3.CMS.git

File: Documentation/Setup/Git/Index.rst [hunk 2]
```diff
@@ -157,7 +157,7 @@ Setting up Your Remote
 
 You must instruct Git to push to Gerrit_ instead of the original repository. It acts as a kind of facade in front of Git::
 
-   git config url."ssh://<YOUR_TYPO3_USERNAME>@review.typo3.org:29418".pushInsteadOf git://git.typo3.org
+   git config url."ssh://<YOUR_TYPO3_USERNAME>@review.typo3.org:29418/Packages/TYPO3.CMS.git"
```
Comment (susannemoog) [inline Documentation/Setup/Git/Index.rst:160]: git config remote.origin.pushurl ssh://<username>@review.typo3.org:29418/Packages/TYPO3.CMS.git

File: Documentation/Setup/Git/Index.rst [hunk 3]
```diff
@@ -234,15 +234,15 @@ Or, compare the :file:`.git/config` file inside the repository::
...
+       url = git://github.com/typo3/typo3
        fetch = +refs/heads/*:refs/remotes/origin/*
     [branch "master"]
        remote = origin
        merge = refs/heads/master
     [branch]
        autosetuprebase = remote
-    [url "ssh://<YOUR_TYPO3_USERNAME>@review.typo3.org:29418"]
-       pushInsteadOf = git://git.typo3.org
+    [url "ssh://<YOUR_TYPO3_USERNAME>@review.typo3.org:29418/Packages/TYPO3.CMS.git"]
```
Comment (susannemoog) [inline Documentation/Setup/Git/Index.rst:244]: remove this.

File: Documentation/Setup/Git/Index.rst [hunk 4]
```diff
@@ -234,15 +234,15 @@ Or, compare the :file:`.git/config` file inside the repository::
        ignorecase = true
        precomposeunicode = true
     [remote "origin"]
-       url = git://git.typo3.org/Packages/TYPO3.CMS.git
+       url = git://github.com/typo3/typo3
        fetch = +refs/heads/*:refs/remotes/origin/*
```
Comment (susannemoog) [inline Documentation/Setup/Git/Index.rst:238]: [remote "origin"]
        url = git@github.com:typo3/typo3
        fetch = +refs/heads/*:refs/remotes/origin/*
        pushurl = ssh://<TYPO3_USER_NAME>@review.typo3.org:29418/Packages/TYPO3.CMS.git

File: Documentation/BugfixingAZ/Index.rst [hunk 5]
```diff
@@ -91,7 +91,7 @@ and setting up a working TYPO3 installation for core development, skip to number
 
    Push to Gerrit (replace `<YOUR_TYPO3_USERNAME>` here)::
 
-      git config url."ssh://<YOUR_TYPO3_USERNAME>@review.typo3.org:29418/Packages/TYPO3.CMS.git"
+      git config remote.origin.pushurl ssh://@review.typo3.org:29418/Packages/TYPO3.CMS.git
```
Comment (susannemoog) [inline Documentation/BugfixingAZ/Index.rst:94]: missing the username

File: Documentation/Setup/Git/Index.rst [hunk 6]
```diff
@@ -157,7 +157,7 @@ Setting up Your Remote
 
 You must instruct Git to push to Gerrit_ instead of the original repository. It acts as a kind of facade in front of Git::
 
-   git config url."ssh://<YOUR_TYPO3_USERNAME>@review.typo3.org:29418".pushInsteadOf git://git.typo3.org
+   git config remote.origin.pushurl ssh://@review.typo3.org:29418/Packages/TYPO3.CMS.git
```
Comment (susannemoog) [inline Documentation/Setup/Git/Index.rst:160]: missing username

**Review by susannemoog — APPROVED** [review]

**Comment by linawolf** [comment]
Releases: master, 10.4, 9.5

**Pull request merged**